python/ray/tune/examples/pbt_transformers/pbt_transformers.py

```diff
@@ -84,28 +82,19 @@ def get_model():
         logging_dir="./logs",
     )
 
+    training_args._n_gpu = gpus_per_trial
+
     trainer = Trainer(
         model_init=get_model,
         args=training_args,
         train_dataset=train_dataset,
         eval_dataset=eval_dataset,
         compute_metrics=build_compute_metrics_fn(task_name))
 
-    # Number of eval steps is dependent on per_device_train_batch_size.
-    # So we define a separate function that takes in a spec arg.
-    def eval_steps_func(spec):
-        if not smoke_test:
-            return len(train_dataset
-                       ) // spec.config["per_device_train_batch_size"] + 1
-        else:
-            return min(1, spec.config["per_device_train_batch_size"])
-
     tune_config = {
         "per_device_train_batch_size": 32,
         "per_device_eval_batch_size": 32,
-        "eval_steps": tune.sample_from(eval_steps_func),
-        "save_steps": tune.sample_from(lambda spec: spec.config["eval_steps"]),
-        "num_train_epochs": tune.choice([2, 3, 4, 5]),
+        "num_train_epochs": 10,
```

Comment:
should we still keep `tune.choice([2, 3, 4, 5])`?

Reply:
Ah yes my bad- I changed it back.